snuba/web/query.py

```diff
@@ -69,17 +39,6 @@ def _run_new_query_pipeline(
             data=request, query_settings=request.query_settings, timer=timer, error=None
         )
     )
-    if force_dry_run:
```

Comment:
how come this goes away?

Reply:
that was only necessary for the dry run rollout. You QuerySettings set dry run here

Reply:
ah I see, we still have dry run right, itll just get set somewhere else now that its rolled out?

Reply:
correct, it's a field in the query settings. We were just forcing it for the rollout